File: scripts/debug_mongo.js
const { MongoClient } = require('mongodb');

async function run() {
    const client = new MongoClient('mongodb://localhost:27017');
    await client.connect();
    const db = client.db('gamedb');

    const game = await db.collection('games').findOne({ name: /Mario/i });
    console.log("Game ID:", game._id, "Name:", game.name);
    console.log("Platforms:", game.platforms);
    console.log("Genres:", game.genres);
    console.log("Themes:", game.themes);
    console.log("Involved Companies:", game.involved_companies);

    await client.close();
}

run().catch(console.dir);

File: scripts/debug_mongo.py
from pymongo import MongoClient
import json

client = MongoClient('mongodb://admin:password@localhost:27017/?authSource=admin')
db = client['gamedb']

# Find games that actually have genres
games = list(db['games'].find({"name": {"$regex": "Mario", "$options": "i"}}).limit(2))
for g in games:
    print("Name:", g.get('name'))
    print("Genres:", g.get('genres'))
    print("Themes:", g.get('themes'))
    print("Platforms:", g.get('platforms'))
    print("---")
